perf/benchmark/runner/runner.py

```diff
@@ -394,10 +388,6 @@ def get_parser():
         "--ingress",
         help="run traffic through ingress",
         default=None)
-    parser.add_argument(
```

Comment:
we need to keep this argument if some other user want to pass in extra labels. you may rename it as --extra_labels?